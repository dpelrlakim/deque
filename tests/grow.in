push_front a 0
push_front a -1
push_front a 2
push_front a -3
push_front a 4
push_front a -5
pop_front a
pop_front a
pop_front a
pop_front a
pop_front a
pop_front a
newDefault b
push_back b 0
push_back b 1
push_back b -2
push_back b 2147483647
push_back b -2147483648
[]= b 3 11
pop_back b
push_back b 27
push_back b 300
[] b 5
pop_back b
pop_back b
pop_back b
pop_back b
pop_back b
pop_back b
