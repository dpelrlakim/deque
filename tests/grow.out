a: 0
a: -1 0
a: 2 -1 0
a: -3 2 -1 0
a: 4 -3 2 -1 0
a: -5 4 -3 2 -1 0
a: 4 -3 2 -1 0
a: -3 2 -1 0
a: 2 -1 0
a: -1 0
a: 0
a:
a:
b:
a:
b: 0
a:
b: 0 1
a:
b: 0 1 -2
a:
b: 0 1 -2 2147483647
a:
b: 0 1 -2 2147483647 -2147483648
a:
b: 0 1 -2 11 -2147483648
a:
b: 0 1 -2 11
a:
b: 0 1 -2 11 27
a:
b: 0 1 -2 11 27 300
300
a:
b: 0 1 -2 11 27 300
a:
b: 0 1 -2 11 27
a:
b: 0 1 -2 11
a:
b: 0 1 -2
a:
b: 0 1
a:
b: 0
a:
b:
